azext_iot/tests/iothub/configurations/test_iot_config_int.py

```diff
@@ -676,6 +677,29 @@ def test_device_configurations(self):
                 expect_failure=True,
             )
 
+            # Validate target condition for configuration
+            target_condition = "from devices.modules where tags.building=9"
+            self.cmd(
+                self.set_cmd_auth_type(
+                    "iot hub configuration test-queries -n {} --target-condition \"{}\"".format(
+                        self.entity_name, target_condition
+                    ),
+                    auth_type=auth_phase,
+                ),
+                expect_failure=False,
```

Comment:
if expect_failure is false, you don't need to specify that input - magic of python and default values

Reply:
ah I see, will update!